player/command.c

```diff
@@ -1336,6 +1336,64 @@ static int mp_property_chapter_metadata(void *ctx, struct m_property *prop,
     return tag_property(action, arg, mpctx->chapters[chapter].metadata);
 }
 
+static int mp_property_track_metadata(void *ctx, struct m_property *prop,
+                                      int action, void *arg)
+{
+    MPContext *mpctx = ctx;
+
+    if (action != M_PROPERTY_KEY_ACTION)
+        return M_PROPERTY_UNAVAILABLE;
+
+    struct m_property_action_arg *ka = arg;
+    bstr key;
```

Comment:
```suggestion
    bstr key = {0};
```
I know the `m_property_split_path` cannot really fail, but still it is nicer to make sure we have valid object just in case.